dependencies.yml

```diff
@@ -217,6 +217,12 @@ javax.inject:
 javax.validation:
   validation-api: { version: '2.0.1.Final' }
 
+org.testng:
+  testng:
+    version: '7.0.0'
+    javadocs:
+      - https://static.javadoc.io/org.testng/testng/7.0.0/
```

Comment:
We do not refer to TestNG API in our public Javadoc, so we do not need this.

Reply:
Thanks for teaching! I will remove this line.